### ISSUE TITLE
Add ps3netsrv

### ISSUE THREAD

**Issue by bahusoid** [WDCommunity/wdpksrc#48]
Any chance to have [ps3netsrv](https://github.com/stefaang/spksrc/tree/master/cross/ps3netsrv) be published on https://bintray.com/tfl/wdpksrc?

Would be very grateful for WD MyCloud Gen2 version of app.

**Comment by stefaang** [comment]
Any success with this?

[ps3netsrv.tar.gz](https://github.com/WDCommunity/wdpksrc/files/3902521/ps3netsrv.tar.gz)

**Comment by stefaang** [comment]
TODO:
Edit `/shares/Volume_1/Nas_Prog/ps3netsrv/target/ps3netsrv-starter.sh`
```
#!/bin/sh

BIN_FILE="${SYNOPKG_PKGDEST}/bin/ps3netsrv"

if [ -r "${CFG_FILE}" ]; then
    . "${CFG_FILE}"
fi

${BIN_FILE} "${PS3_DIR}" 2>&1 &
echo "$!" > "${PID_FILE}"
```
Edit `/shares/Volume_1/Nas_Prog/ps3netsrv/target/var/ps3netsrv.conf`
```
PS3_DIR="/shares/Volume_1/whatever"
```
Then use this to start it.
```
/shares/Volume_1/Nas_Prog/ps3netsrv/start.sh /shares/Volume_1/Nas_Prog/ps3netsrv
```
More TODO: fix stop.sh

**Comment by bahusoid** [comment]
Awesome!!! 
Big thanks for config fixes. I was just going to complain about it :)

I've been able to successfully install  and run it and it's recognized by PS3.  It's properly starting from Web by ticking Run App (so I didn't try to start it from ssh).

Regarding `PS3_DIR` I think by default it should look into `/shares/Volume_1/Public/PS3`

**Comment by stefaang** [comment]
@tonyrayo you were also looking for this, no?

**Comment by bahusoid** [comment]
Status report: Server was working for few days without any issues. So thanks again for quick response!
Are you planning to publish it officially?

> More TODO: fix stop.sh

Not sure what you meant but I see service is properly stopped from Web when Run App is ticked to OFF (checked logs and list of processes) So it seems nothing needs fixing here.

**Comment by stefaang** [comment]
Oh, okay, good to hear it's stable.
The pid file is stored in the `target` directory, while I'd rather store it in `/var/run` to prevent stale pid files on a system crash (e.g. power cycle).

It will probably take a few weeks before I'm taking this up and doing some proper testing... but feel free to send in a MR to my spksrc branch.

**Comment by tonyrayo** [comment]
> 
> 
> @tonyrayo you were also looking for this, no?

Indeed! I'm currently running it as a docker container for ease of upgrade (your release of Docker/Portainer is wonderous =). Apologies for my lengthy absence by the way; health issues mixed with a return to edu has kept me away from development (as well as most things that fall under "free time projects"). I look forward to hopping back in, even if as a minor role at first!
